3.10/Dockerfile

```
@@ -32,7 +34,7 @@ RUN sudo apt-get update && sudo apt-get install -y \
 	curl https://pyenv.run | bash && \
 	sudo rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-RUN env PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install 3.10.1 && pyenv global 3.10.1
+RUN env PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install 3.10.2 && pyenv global 3.10.2
```

Can we add this flag to `Dockerfile.template` along with upgrading the versions?

```suggestion
RUN env PYTHON_CONFIGURE_OPTS="--enable-shared --enable-optimizations" pyenv install 3.10.2 && pyenv global 3.10.2
```

See #108

---

Thank you for the nudge. Your opened GitHub Issue has been completed and merged in, and we'll be adding the change to this PR.

---

Thank you! This should be a noticeable enhancement.